README.markdown

```diff
@@ -309,7 +295,7 @@ To enable a report of application instances in the DB, you'll need to define the
 1. Create the test database. You only need to do this once:
 
    ```bash
-   $ psql postgresql://postgres@localhost:5433/postgres -c "CREATE DATABASE lms_test;"
+   $ docker-compose exec postgres psql -U postgres -c "CREATE DATABASE lms_test;"
```

Comment:
I'll send PRs to get rid of this annoying step from both h and lms later. If the tests use a separate `postgres_test` container instead of reusing the dev server's `postgres` container, then they should be able to use the default `postgres` DB in that container and not have to create a custom DB. And `docker-compose` will handle creating this container automatically for us. (Automating the `CREATE DATABASE` command is awkward b/c Postgres doesn't provide a good "Create named DB if it doesn't exist, and don't fail if it does" command, workarounds are possible but awkward.)